### PR TITLE
NOTICK: detect bundles without symbolic names and filter them out to avoid later NPEs

### PR DESCRIPTION
**Pull request by dickon**

### CODE REVIEW

File: osgi-framework-bootstrap/src/main/java/net/corda/osgi/framework/OSGiFrameworkWrap.java [hunk 1]
```diff
@@ -291,17 +291,15 @@ OSGiFrameworkWrap activate() throws BundleException {
...
                 logger.info("OSGi bundle {} ID = {} {} {} {} fragment.",
-                    bundle.getLocation(),
-                    bundle.getBundleId(),
-                    symbolicName == null ? "-" : symbolicName,
-                    bundle.getVersion(),
-                    bundleStateMap.get(bundle.getState())
+                        bundle.getLocation(),
+                        bundle.getBundleId(), bundle.getSymbolicName(),
+                        bundle.getVersion(),
+                        bundleStateMap.get(bundle.getState())
```
Comment (chrisr3) [inline osgi-framework-bootstrap/src/main/java/net/corda/osgi/framework/OSGiFrameworkWrap.java]: I really wish IntelliJ wouldn't frig with whitespace, but why is `bundle.getSymbolicName()` on the same line as `bundle.getBundleId()`?

**Review by chrisr3 — COMMENTED** [review]
Looks OK, apart from`
```java
bundle.getBundleId(), bundle.getSymbolicName(),
```
which should be
```java
bundle.getBundleId(),
bundle.getSymbolicName(),
```
for consistency.

File: osgi-framework-bootstrap/src/main/java/net/corda/osgi/framework/OSGiFrameworkWrap.java (hunk 1 above)
Comment (dickon) [reply osgi-framework-bootstrap/src/main/java/net/corda/osgi/framework/OSGiFrameworkWrap.java]: I fixed the formatting and undid the whitespace changes.

**Review by chrisr3 — APPROVED** [review]
LGTM (And _relax!_) :relaxed:

**Pull request merged**